src/test/resources/pages/adminActivityLogPage.html

```diff
@@ -671,11 +552,11 @@ <h4 class="text-center">
 
 
 
-    <div class="alert alert-warning" id="statusMessage">
+    <div class="alert alert-info" id="statusMessage">
           Total Logs gone through in last search: {*}<br /><input type="hidden" value="false" id="ifShowAll" /><input type="hidden" value="false" id="ifShowTestData" /></div>
 
                 
-            {*}
+            <br />
```

Comment:
I don't agree with that, but then again I may understand it wrongly. The 'Older Logs' thing appears in status message. Here's this section of the HTML in the jsp file:
![image](https://cloud.githubusercontent.com/assets/7261051/9603788/b5a6fa5e-50e5-11e5-87c7-cfaf62c620a9.png)
I don't see any reason why `<br>` there needs to be wildcarded. If anything, it should be ` Total Logs gone through in last search: {*}<br>{*}`, which both of us missed.